javascript/javascript2/week2/hyfBay/myHyf.js

```diff
@@ -0,0 +1,115 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+//render products
+function renderProducts(products) {
+    products.forEach(product => {
+        let myDiv = document.createElement("div")
+        myDiv.classList.add("li-container");
+        let myProduct = document.createElement("li");
+        let myPrice = document.createElement("li");
+        let myRating = document.createElement("li");
+        let myUl = document.querySelector(".my-ul");
```

Comment:
Every time `renderProducts` is run, this is run. `let myUl = document.querySelector(".my-ul");`

`querySelector` can be an expensive call so call it once outside the function and reuse `myUl`.

Reply:
Thank you very much for the comment. I'm making the change.